clients/cobol-lsp-vscode-extension/src/services/ProcessorGroupsLoader.ts

```diff
@@ -20,7 +20,17 @@ import { TextDecoder } from "util";
 const PG_FOLDER = ".cobolplugin";
 const PGR_PGM_FILE = "pgm_conf.json";
 const PG_PROC_FILE = "proc_grps.json";
+const DEFAULT_PROCESSOR_GROUP_CONFIG = "[]";
+const DEFAULT_PROCESSOR_CONFIG = '{ "pgms": [] }';
+type WorkspaceConfigEntry =
+  | [string | undefined, string | undefined]
+  | undefined;
+const workspaceConfiguration: Map<string, WorkspaceConfigEntry> = new Map();
```

Comment:
I think the code would be much simplified if it was structured the other way around, not as map of pairs, but rather pair of maps.